src/azul/terraform.py

```diff
@@ -776,6 +781,34 @@ def tf_config(self, app_name):
             resource['source_code_hash'] = '${filebase64sha256("%s")}' % package_zip
             resource['filename'] = package_zip
 
+        # Replace any references to unqualified function ARNs emitted by Chalice
+        # with references to the alias. This must be happen before the injection
+        # of the alias resources, else the aws_lambda_alias.function_name fields
+        # would be affected by the regex substitution.
+        #
+        def function_to_alias(j: MutableJSON) -> MutableJSON:
+            s = json.dumps(j)
+            s = re.sub(r'\$\{aws_lambda_function\.([^.]+)\.(arn|invoke_arn)\}',
```

Comment:
```suggestion
            s = re.sub(r'\$\{aws_lambda_function\.([^.]+)\.(arn|invoke_arn)}',
```

Pretty sure `}` has no special meaning unless preceded by a `{`